cpp/include/cudf/detail/utilities/element_argminmax.cuh

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+/**
+ * @brief Definition of the device operators
+ * @file
+ */
```

Comment:
```suggestion

```
I think this is vague and unnecessary.

Reply:
Agreed, they are remnants when we are copying from https://github.com/rapidsai/cudf/blob/ab801f143fa612c7698737af4125308406c1cff5/cpp/include/cudf/detail/utilities/device_operators.cuh#L19-L22